keywords/couchbaseserver.py

```diff
@@ -247,6 +207,50 @@ def wait_for_ready_state(self):
             # All nodes are heathy if it made it to here
             break
 
+    def _create_internal_rbac_bucket_user(self, bucketname):
```

Comment:
pylint is complaining that this method name is invalid, may be because it starts with _ . Not a big deal, just letting you know.